back/app/models/app_configuration.rb

```diff
@@ -136,6 +137,16 @@ def add_missing_features_and_settings
     self.settings = ss.add_missing_settings(settings, schema)
   end
 
+  def sanitize_organization_name
+    organization_name = settings.dig('core', 'organization_name')
+    return unless organization_name.is_a?(Hash) && organization_name.any?
+
+    settings['core']['organization_name'] = SanitizationService.new.sanitize_multiloc(
+      organization_name,
+      []
```

Comment:
It's not clear to me what a `feature` is in this context. Are you sure it's not too restrictive? Probably fine since we don't use a rich editor for the organization name.

Reply:
I didn't touch the `feature_activated?` method and it is not involved in the sanitization I have added.

Reply:
I'm talking about the `[]` argument :)

Reply:
Oh, I think you mean the `features` in the `def sanitize(text, features)`.

They are the types of html tags that are permitted (non-permitted will be removed). I'll add a comment ro explain this where there is the Todo comment.